model/serverSelect.js

```diff
@@ -1,16 +1,36 @@
 module.exports = function(app, io){
 
+	//Testing values
```

Comment:
To the best of my knowledge, models are generally used to interface directly with a database, so they might export a method that get's data from a db ect ect. And u simply require them to access a database. I could definetly be wrong though.

Instead of having all ur logic in the model, i'd make models have generic db logic, and then creates a folder for http routes and a folder for socket routes. 

So for example say you hit `/player/create` then it would be found in `server/routes/player`
In this example you might have `server/socketRoutes/server`.

This is a high level kind of comment, talk to me more about it if you want